src/Command/ImageCopyCommand.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace RokkaCli\Command;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ImageCopyCommand extends BaseRokkaCliCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('image:copy')
+            ->setDescription('Copy the given image to another organization')
+            ->addArgument('hash', InputArgument::REQUIRED, 'The Source Image hash')
+            ->addArgument('destination', InputArgument::REQUIRED, 'The destination organization')
+            ->addOption('organization', null, InputOption::VALUE_REQUIRED, 'The organization to delete the images from')
```

Comment:
erm, why delete? should we call this option `source` instead? and mention that its for when you want to copy from a non-default organization?

Reply:
it's organization in all other options with the almost the same meaning. I'd leave it like that.
The delete was a copy&paste mistake

Reply:
fair enough, lets keep it consistent then.

Reply:
Just realized the image:clone command used it differently. 
```
            ->addArgument('dest-organization', InputArgument::REQUIRED, 'The destination organization to copy images to')
            ->addOption('source-organization', null, InputOption::VALUE_REQUIRED, 'The source organization to copy images from', null)
```

I'll therefore will use that for image:copy as well